crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1758,6 +1759,14 @@ where
                             signed_effects.into_data(),
                             cert_sig,
                         );
+
+                        if (state.request.include_input_objects && state.input_objects.is_none())
+                            || (state.request.include_output_objects
+                                && state.output_objects.is_none())
+                        {
+                            debug!(?tx_digest, "Quorum Reached but requested input/output objects were not returned");
```

Comment:
Could we use a metric? debug logs will be very hard to dig. Feel free to do it in a separate PR though

Reply:
Will add a metrics as well, thanks for the suggestion